README.adoc

```diff
@@ -2523,7 +2523,14 @@ validates :name, presence: true
 
 ==== Methods That Have "keyword" Status in Ruby [[method-invocation-parens-keyword]]
 
-Always omit parentheses for methods that have "keyword" status in Ruby.
+The set of methods that have "keyword" status is not well-defined.
+There is agreement that declarative methods have "keyword" status, but which methods are declarative methods is not well-defined.
```

Comment:
I'm not sure how common the term "declarative" is in Ruby. More often than not I've seen those called DSL methods or macro methods. Might make sense to mention all terms for clarity.

Reply:
OK, I force-pushed an amended commit that adds the other terms for clarity.